src/backend/prisma/migrations/20230702055640_init/migration.sql

```diff
@@ -0,0 +1,30 @@
+-- CreateTable
+CREATE TABLE "project" (
+    "id" TEXT NOT NULL,
```

Comment:
Any reason not to do it as `"id" TEXT NO NULL PRIMARY KEY, `?

Reply:
![image](https://github.com/Sasporta/projects-manager/assets/69085015/082c2442-a7aa-47ca-85e4-5161f39a5953)
It's there at the bottom.